build/stringify.js

```diff
@@ -9,7 +9,7 @@ const isPrimitive = obj => obj === null || ['string', 'number', 'boolean'].inclu
 const isArrayOfPrimitive = obj => Array.isArray(obj) && obj.every(isPrimitive)
 const format = arr => `^^^[ ${arr.map(val => JSON.stringify(val)).join(', ')} ]`
 const replacer = (key, value) => isArrayOfPrimitive(value) ? format(value) : value // objectsort(value)
-const expand = str => str.replace(/"\^\^\^(\[ .* \])"/g, (match, a) => a.replace(/\\"/g, '"')).replace(/\\\\"/g, "'")
-const stringify = obj => expand(JSON.stringify(Array.isArray(obj) ? dedupe(obj) : obj, replacer, 2))
+const expand = str => str.replace(/"\^\^\^(\[ .* ])"/g, (match, a) => a.replace(/\\"/g, '"')).replace(/\\\\"/g, "'")
+const stringify = (obj) => expand(JSON.stringify(Array.isArray(obj) ? dedupe(obj) : obj, replacer, 2))
```

Comment:
I think it's better to remove the parentheses to keep the project consistent
```suggestion
const stringify = obj => expand(JSON.stringify(Array.isArray(obj) ? dedupe(obj) : obj, replacer, 2))
```

Reply:
I'm planning a styling update pass before I do the ES upgrade. Can I do it then?

Reply:
Sure if you plan to change it anyway it won't be a problem.